examples/src/rescue/rescue.rs

```diff
@@ -45,6 +45,14 @@ pub fn apply_round(state: &mut [BaseElement], step: usize) {
     add_constants(state, &ark, STATE_WIDTH);
 }
 
+pub fn apply_round_parallel(multi_state: &mut [BaseElement], step: usize) {
+    debug_assert_eq!(multi_state.len() % STATE_WIDTH, 0);
+
+    for state in multi_state.chunks_mut(STATE_WIDTH) {
+        apply_round(state, step)
+    }
+}
+
```

Comment:
This function is specific for the `rescue_rap` example - maybe we should move it there?